mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -825,6 +997,9 @@
         <CostResource index="Minerals" value="15"/>
         <ScoreMake value="15"/>
         <ScoreKill value="15"/>
+        <ShieldsStart value="30"/>
+        <ShieldsMax value="30"/>
+        <AttackTargetPriority value="19"/>
```

Comment:
Interceptor attack target priority reduced from 20 to 19

This makes it be prioritized less than usual attacking units and spell casters. When units need a new target to fire upon, they will not shoot at Interceptors unless there is no other combat unit around anymore